data/categories.txt

```diff
@@ -1962,7 +1962,7 @@ nl:3^Veerpont|veerboot|veer|pont|overzetboot
 fi:3^Lautta|terminaali|kuljetus
 fr:3^Traversier|terminal
 de:2^Fähre|Terminal
-hu:2^Komp
+hu:2^Kompkikötő
```

Comment:
Is it really so rare (or important) to start suggesting it after typing "Ko" (2 characters)?

Reply:
No, sorry, I just took over from the previous version.